functions/Import-DbaCsv.ps1

```diff
@@ -515,6 +519,18 @@ function Import-DbaCsv {
                         $bulkCopy.NotifyAfter = $NotifyAfter
                         $bulkCopy.EnableStreaming = $true
 
+                        if ($AutoMapColumns) {
```

Comment:
Thank you this is great! I considered something in the past, but didnt include it because my default was to do it by default and it doesn't work as often as not mapping somehow?

This looks nice, except `if ($AutoMapColumns) {` should be `if ($AutoColumnMap) {` yes?

Reply:
yup, should prob have done a bit more testing & noticed that. I imagine it snuck by while I was importing/re-importing/editting just before I pushed. 

tbh, I'm not really familiar with how the appveyor pester for this function works (where is `SuperSmall.csv` saved for example, and how to define a target schema within the appveyor db against which to test this) but I'd be happy to do a bit more build out if you wanna slap a <kbd>DO NOT MERGE</kbd>🏷  on this and give me a couple days to figure it out 😄